### PR TITLE
adding api diff for preview 5 and preview 6

### PR DESCRIPTION
**Pull request by Anipik**
cc @safern @tarekgh @krwq @carlossanlop @jozkee @AaronRobinsonMSFT @jkoritzinsky @layomia @steveharter @layomia  @tommcdon  @buyaa-n  @GrabYourPitchforks @dotnet/ncl @tannergooding @danmosemsft @richlander 

@mkArtakMSFT @pranavkm @halter73 @Tratcher @BrennanConroy @anurse

### CODE REVIEW

File: release-notes/5.0/preview/api-diff/preview6/.Net/5.0-preview6_System.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# System
+
+``` diff
+ namespace System {
+     public static class Math {
++        public static long BigMul(long a, long b, out long low);
++        public static ulong BigMul(ulong a, ulong b, out ulong low);
```
Comment (tannergooding) [inline release-notes/5.0/preview/api-diff/preview6/.Net/5.0-preview6_System.md:7]: Math additions LGTM. CC. @pgovind

File: release-notes/5.0/preview/api-diff/preview5/.Net/5.0-preview5_System.Runtime.Intrinsics.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# System.Runtime.Intrinsics
+
+``` diff
+ namespace System.Runtime.Intrinsics {
```
Comment (tannergooding) [inline release-notes/5.0/preview/api-diff/preview5/.Net/5.0-preview5_System.Runtime.Intrinsics.md:4]: System.Runtime.Intrinsics changes LGTM. CC. @CarolEidt, @echesakovMSFT 

The next preview (preview 7) should have all of ARM moved in box.

**Comment by Tratcher** [comment]
:shipit: for the rest of AspNetCore.

**Comment by tommcdon** [comment]
LGTM. Adding @sywhang in case there are perf counter API's

**Comment by tarekgh** [comment]
System.Globalization and System.Text looks good to me. @GrabYourPitchforks can take a quick look too at these.

File: release-notes/5.0/preview/api-diff/preview6/.Net/5.0-preview6_System.Text.Json.Serialization.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
++        protected ReferenceHandler();
++        public static ReferenceHandler Preserve { get; }
++        public abstract ReferenceResolver CreateResolver();
++    }
++    public sealed class ReferenceHandler<T> : ReferenceHandler where T : ReferenceResolver, new() {
++        public ReferenceHandler();
++        public override ReferenceResolver CreateResolver();
++    }
+-    public sealed class ReferenceHandling {
+ {
```
Comment (Jozkee) [inline release-notes/5.0/preview/api-diff/preview6/.Net/5.0-preview6_System.Text.Json.Serialization.md:24]: I think this is an extra bracket.

File: release-notes/5.0/preview/api-diff/preview5/.Net/5.0-preview5_System.Xml.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# System.Xml
+
+``` diff
+ namespace System.Xml {
+-    public abstract class XmlWriter : IDisposable {
++    public abstract class XmlWriter : IAsyncDisposable, IDisposable {
```
Comment (buyaa-n) [inline release-notes/5.0/preview/api-diff/preview5/.Net/5.0-preview5_System.Xml.md:6]: XML changes LGTM

**Comment by sywhang** [comment]
> Adding @sywhang in case there are perf counter API's

Confirming there aren't any new EventCounter related APIs added between Preview 5 and 6. Thanks.

**Comment by layomia** [comment]
Aside from @Jozkee's comment https://github.com/dotnet/core/pull/4868#discussion_r445718288, System.Text.Json changes LGTM

**Comment by AaronRobinsonMSFT** [comment]
InteropServices signed off.

**Comment by mkArtakMSFT** [comment]
@pranavkm , @javiercn can one of you please look into the changes in the relevant areas? Thanks!

File: release-notes/5.0/preview/api-diff/preview5/.Net/5.0-preview5_System.IO.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+# System.IO
+
+``` diff
+ namespace System.IO {
```
Comment (carlossanlop) [inline release-notes/5.0/preview/api-diff/preview5/.Net/5.0-preview5_System.IO.md:4]: LGTM System.IO

**Pull request merged**